core/app/c/[communitySlug]/types/TypeEditor.tsx

```diff
@@ -172,7 +172,7 @@ export const TypeEditor = ({ onTypeCreation }: Props) => {
 														</td>
 														<td>
 															<FormItem
-																data-testId={`${pubField.slug}-titleField`}
+																data-testid={`${pubField.slug}-titleField`}
```

Comment:
i added this incorrect attribute in a pr a few months ago which was messing up tests